### PR TITLE
More consistent / thicker header icons and spacing.

### PR DESCRIPTION
**Pull request by Billiam**
I increased the font size of the studio header to give the icons a little more room, but also resized some of the icon so that they fill the space more consistently, and made a go of standardizing the overall line widths between them.

It's not 100%, but I think it's an improvement for readability.

Studio Before/After:
image
image

Visualizer:
image
image

### CODE REVIEW

**Review by sgenoud — APPROVED** [review]
Thanks! I might have a go at it after you merge this (can you merge actually, not sure how github works by default for this?).

If you want to play with the last icon remaining for the share screen you can open `/share/test-shape` to play with the share visualiser.

Thanks for the series of changes on the studio! It feels great to have a series of improvements! Hope getting into this code wasn't too much of a headache!

**Comment by Billiam** [comment]
> Thanks! I might have a go at it after you merge this (can you merge actually, not sure how github works by default for this?).

I cannot! I believe only people with write access to the repository would be able to merge it, though I think there are some options for automatic merging in github somewhere.

> If you want to play with the last icon remaining for the share screen you can open `/share/test-shape` to play with the share visualiser.

Is that the 'focus' icon? I think it looks alright currently because of the thicker center dot, but I'm not sure

image

> Hope getting into this code wasn't too much of a headache!

Not at all, everything's pretty much where I'd expect :ok_hand: 

The only thing I might suggest is adding setup steps to the readme for contributors to get up and running (pnpm install, prepare-packages, pnpm start).

**Pull request merged**
